docs/topics/running/environment.md

```diff
@@ -62,32 +62,71 @@ verbose, valid log levels are `error`, `warn`/`warning`, `info`,
 
-If `REDIS_PASSWORD` (or `REDIS_PERSECOND_PASSWORD`) is non-empty, then
-it is used to `AUTH` to Redis immediately after the connection is
-established.  If `REDIS_USERNAME` (or `REDIS_PERSECOND_USERNAME`) is
-set, then that username is used with to log in as that user in the
-[Redis 6 ACL].  It is invalid to set a username without setting a
-password.  It is invalid to set a username with Redis 5 or lower.
-
-If `REDIS_TLS_ENABLED` (or `REDIS_PERSECOND_TLS_ENABLED`) is true,
-then TLS is used when communicating with Redis.  Setting
-`REDIS_TLS_INSECURE` (or `REDIS_PERSECOND_TLS_INSECURE`) to true
-disables TLS verification when doing so; alternatively, consider
-[installing Redis' self-signed certificate in to the Ambassador Edge
-Stack
-container](../../using/filters/#installing-self-signed-certificates).
-
-Note that when using a port name instead of a port number in a Go
-network address (as as in `REDIS_URL` or `REDIS_PERSECOND_URL`), the
-name refers a well-known port name in the container's `/etc/services`,
-and **not** to a Kubernetes port name.
+#### Redis layer 4 connectivity
+
+- `SOCKET_TYPE` and `URL` are the Go network name and Go network
+  address to dial to talk to Redis; see [Go `net.Dial`][].  Note that
+  when using a port name instead of a port number, the name refers a
+  well-known port name in the container's `/etc/services`, and **not**
+  to a Kubernetes port name.
+- `TLS_ENABLED` (new in 1.5.0) specifies whether to use TLS when
+  talking to Redis.
+- `TLS_INSECURE` (new in 1.5.0) specifies whether to skip certificate
+  verification when using TLS to talk to Redis.  Alternatively,
+  consider [installing Redis' self-signed certificate in to the
+  Ambassador Edge Stack
+  container](../../using/filters/#installing-self-signed-certificates)
+  in order to leave certificate verification on.
+
+#### Redis authentication
+
+- If `PASSWORD` (new in 1.5.0) is non-empty, then it is used to `AUTH`
+  to Redis immediately after the connection is established.
+- If `USERNAME` (new in 1.5.0) is set, then that username is used with
+  the password to log in as that user in the [Redis 6 ACL].  It is
+  invalid to set a username without setting a password.  It is invalid
+  to set a username with Redis 5 or lower.
+
+#### Redis performance tuning
+
+- `POOL_SIZE` is the number of connections to keep around when idle.
+  The total number of connections may go lower than this if there are
+  errors.  The total number of connections may go higher than this
+  during a load surge.
+- `PING_INTERVAL` (new in 1.6.0) Of the idle connections in the normal
+  pool (not extra connections created for a load surge), Ambassador
+  will `PING` one of them every `PING_INTERVAL` seconds; each
+  connection will on average be `PING`ed every
+  `PING_INTERVAL×POOL_SIZE` seconds; increasing `POOL_SIZE` without
+  reducing `PING_INTERVAL` will increase the amount of time between
+  `PING`s on a given connection.  (Backward incompatibility: prior to
```

Comment:
The wording is a bit strange to me.  Maybe "prior to 1.6.0, `PING_INTERVAL` is set at `PING_INTERVAL=10s÷POOL_SIZE` and cannot be specified.

Reply:
How's "prior to the introduction of this setting in 1.6.0 it was non-configurable  and would `PING` a connection every `10÷POOL_SIZE` seconds, as if  `PING_INTERVAL=10÷POOL_SIZE`."?

Reply:
That works for me